projects/go-lib/src/lib/components/go-checkbox/go-checkbox.component.ts

```diff
@@ -1,24 +1,41 @@
-import { Component, Input, OnInit } from '@angular/core';
+import { AfterViewInit, Component, ElementRef, EventEmitter, Input, OnChanges, OnInit, Output, ViewChild } from '@angular/core';
 import { FormControl } from '@angular/forms';
 import { generateId } from '../../utilities/form.utils';
 
 @Component({
   selector: 'go-checkbox',
   templateUrl: './go-checkbox.component.html'
 })
-export class GoCheckboxComponent implements OnInit {
+export class GoCheckboxComponent implements OnInit, OnChanges, AfterViewInit {
   id: string;
 
   @Input() control: FormControl;
   @Input() disabled: boolean = false;
   @Input() hints: string[];
+  @Input() indeterminate: boolean = false;
   @Input() key: string;
   @Input() label: string;
   @Input() theme: string = 'light';
 
+  @Output() checkboxChange: EventEmitter<boolean> = new EventEmitter<boolean>();
```

Comment:
@StevenUlmer I don't love that I had to add this, but in the `go-table` template we need to be able to pass the row along with the value when a row checkbox value changes to the `selectChange` method. I couldn't think of another way to be able to know which row the checkbox being clicked is in. Open to other ideas if you have any.

Reply:
@jaredami Instead of doing an output here, can we listen do `control.valueChanges.subscribe` in the table for each checkbox? That would prevent us from having to add this here and would be closer to what we've been doing in other places when we need to know if an input has been changed